packages/rum-recorder/src/domain/record/serializationUtils.ts

```diff
@@ -125,5 +126,27 @@ export function getElementInputValue(element: Element, ancestorInputPrivacyMode?
 }
 
 export function maskValue(value: string) {
+  if (isFlagEnabled('privacy-by-default-poc')) {
+    return value.replace(/.+/, PRIVACY_INPUT_MASK)
```

Comment:
FMU, you are trying to replace the value with `PRIVACY_INPUT_MASK`, right? So the value length doesn't leak.

First, what about:
```suggestion
    return PRIVACY_INPUT_MASK
```

This is not enough to avoid leaking input length, because we are capturing every keystroke, so we will generate one `IncrementalSnapshot.Input` record for each character entered by the user. We might want to adjust things [around here](https://github.com/DataDog/browser-sdk/blob/90573dcabdefe68b6c1817d463c8b73f60820355/packages/rum-recorder/src/domain/record/observer.ts#L219).

But, I think we should not try to implement this in the same PR.

Reply:
Agreed. More to discuss, probably making a distinction between `block` and `mask` (or other synonyms) whereby `mask` censors (or at least throttles) JS events. I imagine `block` makes more sense for simple stuff like text nodes (`textContent`)